src/helpers/common.ts

```diff
@@ -1,4 +1,6 @@
 import { z } from 'zod';
+import type { Callback } from '../client.js';
```

Comment:
unused import

Reply:
Will update the Github Action in separate PR to detect this.